Readme.md

```diff
@@ -32,4 +36,57 @@ A [seedr.cc](https://www.seedr.cc/) inspired web application to download torrent
     </tr>
 </table>
 
-</html>
\ No newline at end of file
+</html>
+
+# How to set-up
+
+This section tells you how to setup a self-hosted instance of Autolycus
+
+## Pre-requisites
+
+You will need the following before installation:
+
+1. Docker Command line tools
+    1. [Docker Engine](https://docs.docker.com/engine/install)
+    1. [Docker Compose](https://docs.docker.com/compose/install/) command
+1. Postgresql database with its username, password and host
```

Comment:
Users don't need to have already running Postgres instance, `docker-compose` will take care of it.

Reply:
Oh...

Reply:
Please remove this line
````
Postgresql database with its username, password and host
````